src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -5457,6 +5457,217 @@ public class B : A
             await TestInRegularAndScriptAsync(testText, expected);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public Task TestRefactoringFieldFromModifier()
+        {
+            var testText = @"
+class Base
+{
+}
+
+class Derived : Base
+{
+    pub[||]lic int TestInt;
```

Comment:
Can you add a test where the field has an initializer as well?

Reply:
This is a good point, I originally think initializer should not trigger the refactoring but now I feel why not. This will simplify the problem a lot